### DISCUSSION TITLE
Taiko Full Alt Lazer Mod Request

### DISCUSSION THREAD

**Discussion by maxie5353** [ppy/osu#23131] [Ideas]
Not too sure if this is the place to request this, if this is the incorrect area please let me know where it should be submitted.

The request is to add a full alt mod for taiko to osu lazer. This would function similarly to the standard mod "alternate" which already exists. 

The mod would make it so if that if you press the button for a left kat you would be not be able to press a left don until you hit either a right don or a right kat. Regarding finishers they can be made as a reset where after double tapping you can start the next pattern on either the left or right side.

Again sorry if this is the wrong place to request this stuff, I do not use github so I hope this was formatted and submitted properly.

**Comment by bdach** [comment]
This was attempted but got stuck in kddkkdkkddkdk hell. For brave readers: https://github.com/ppy/osu/pull/9606

**Reply by 404user-not-found** [reply]
1. is there going to be another attempt?
2. is there any working version of code for kddk?

edit: been trying to find something that will force me to full alt stuff for 30 months

**Comment by Nunolin** [comment]
Is this ever getting attempted again? I feel like it's such a good idea especially considering how difficult it is to be mindful of if you're actually alternating with the KDDK full alt playstyle

**Comment by wwawaawwa** [comment]
i would love to see this implemented, even if initially in some rudimentary form for only kddk since that seems like the most straightforward to do
